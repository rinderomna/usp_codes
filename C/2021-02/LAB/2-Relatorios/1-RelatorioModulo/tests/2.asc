50
-93346 -90595 -88011 -84364 -83461 -81320 -75751 -74713 -72577 -68252 -58638 -53676 -45922 -36150 -33708 -33299 -31482 -26182 -3114 402 787 9845 11787 11969 12329 17137 22954 25422 27150 28716 34567 35899 38758 40795 43601 44404 48596 52313 54888 59795 61279 65637 68317 75519 78284 81425 85318 89336 97512 99557
