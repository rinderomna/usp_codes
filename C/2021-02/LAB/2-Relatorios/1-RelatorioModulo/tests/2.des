50
99557 97512 89336 85318 81425 78284 75519 68317 65637 61279 59795 54888 52313 48596 44404 43601 40795 38758 35899 34567 28716 27150 25422 22954 17137 12329 11969 11787 9845 787 402 -3114 -26182 -31482 -33299 -33708 -36150 -45922 -53676 -58638 -68252 -72577 -74713 -75751 -81320 -83461 -84364 -88011 -90595 -93346
