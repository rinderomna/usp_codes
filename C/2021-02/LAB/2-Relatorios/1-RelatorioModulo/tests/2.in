50
-84554 -62578 -27235 8133 85126 -15046 -71946 -96121 -14577 54548 -2399 89386 -46940 13713 93093 -5265 -18891 -33066 35686 -82595 -2776 -10973 -62627 53071 12508 12162 75519 -27838 -86001 62782 21779 6069 -53878 -76385 63660 -45188 -99557 15093 -11647 38936 -84086 -1455 -24182 6228 -59940 99666 -49342 99592 95586 -52114
