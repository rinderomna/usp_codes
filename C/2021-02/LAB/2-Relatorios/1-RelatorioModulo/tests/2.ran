50
-81320 34567 75519 402 78284 -72577 52313 -53676 -84364 -83461 -58638 48596 38758 -3114 85318 99557 -36150 9845 68317 35899 97512 40795 28716 -33299 -90595 44404 -74713 -26182 27150 -75751 61279 89336 12329 -45922 -33708 65637 59795 -31482 43601 -93346 787 11787 -68252 54888 17137 81425 -88011 11969 22954 25422
