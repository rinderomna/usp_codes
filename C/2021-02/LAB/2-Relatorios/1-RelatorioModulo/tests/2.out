-99557 -96121 -86001 -84554 -84086 -82595 -76385 -71946 -62627 -62578 -59940 -53878 -52114 -49342 -46940 -45188 -33066 -27838 -27235 -24182 -18891 -15046 -14577 -11647 -10973 -5265 -2776 -2399 -1455 6069 6228 8133 12162 12508 13713 15093 21779 35686 38936 53071 54548 62782 63660 75519 85126 89386 93093 95586 99592 99666
